### ISSUE TITLE
Files containing space do in name do not show up

### ISSUE THREAD

**Issue by gringofe** [EliuX/MEGAcmd4J#42]
When calling `MegaSession#ls`, files containing a space in their name do not show up

## Expected Behavior
Files with space in their name should show up

## Current Behavior
Files with space in their name do not show up

## Steps to Reproduce
1. Create a file with space in it's name, and one without a space, and upload it to Mega (e.g. `touch test123.txt && touch "test 123.txt" && mega-put test\ 123.txt test123.txt /`)
2. Confirm both show up (e.g. `mega-ls -la /`)
3. Run sample code:
```
public class Test {
    public static void main(String[] args) {
        MegaSession session = Mega.init();
        session.ls("/").call().forEach(fi -> {
            System.out.println(fi.getName());
        });
    }
}
```
4. Only the file without a space in its name shows up

**Comment by EliuX** [comment]
Hi @gringofe, thanks again for your reports and your patience.

Regarding to listing the filename with an space with it: there is actually a test that verifies that:
```
  @DisplayName("Move file from/to remote path with whitespaces")
  @Tag("ls")
  @Tag("mv")
  @Order(15)
  @Test()
  public void shouldMoveFileFromPathWithWhitespace() {
    sessionMega.move("megacmd4j/remote folder/yolo.txt",
        "megacmd4j/level2/another folder/yolo moved.txt")
        .run();

    assertTrue(sessionMega.ls(
        "megacmd4j/level2/another folder/yolo moved.txt"
    ).exists());
  }
```

But I wanted to reproduce exactly your issue, so I created this a test to verify whether it passes with your exact conditions:
```
  @DisplayName("Move file from/to remote path with whitespaces")
  @Test()
  public void shouldListFileWithSpaceInTheRootFolder() {
    MegaTestUtils.createTextFile(
        "target/test 123.txt",
        "Content of file with space"
    );
    MegaTestUtils.createTextFile(
        "target/test123.txt",
        "Content of file without space"
    );

    sessionMega.uploadFiles("/", "target/test123.txt", "target/test 123.txt")
        .run();

    final List<FileInfo> files = sessionMega.ls("/").call();

    Assert.assertTrue(
        files.stream().filter(f -> f.getName().equals("test123.txt")).findAny().isPresent()
    );
    Assert.assertTrue(
        files.stream().filter(f -> f.getName().equals("test 123.txt")).findAny().isPresent()
    );
    Assert.assertFalse(
        files.stream().filter(f -> f.getName().equals("test123123123.txt")).findAny().isPresent()
    );
  }
```
![image](https://user-images.githubusercontent.com/6514740/164997763-c9d85565-586a-4ea8-b129-bc958a89faa2.png)

So please verify what is happening in your side but this should work without any problem. Nonetheless, I will add this new test to verify your usecase.

**Comment by EliuX** [comment]
Hi @gringofe, as always, thanks for your feedback and your patience. 
I was able to reproduce your error, and I got a solution for you. I hope it helps.

You will see it fixed in the version `v1.6.11`:
```xml
<dependency>
  <groupId>com.github.eliux</groupId>
  <artifactId>megacmd4j</artifactId>
  <version>1.6.11</version>
</dependency>

```
